webpack.config.js

```diff
@@ -116,7 +117,8 @@ module.exports = {
   },
 
   optimization: {
-    minimize: false
+    minimize: process.env.NODE_ENV === 'production',
```

Comment:
This should not be needed.  As of webpack 4, production builds are automatically minimized.

Reply:
good point, that's just a holdover from when i was forcing minimize on dev env